### ISSUE TITLE
Compare with tracecode

### ISSUE THREAD

**Issue by zvr** [eellak/build-recorder#154]
During the FOSDEM presentation, there was a question about performance comparison to [tracecode](https://github.com/nexB/tracecode-toolkit-strace).

We should research this and add some documentation about this.

**Comment by zvr** [comment]
OK, from a very quick first look, `tracecode` seems to be scripts processing the output of `strace -ff -y -ttt -qq -a1`.

**Comment by fvalasiad** [comment]
@zvr Yes i looked at it too, which means that it cannot be optimized for builds or do other specific tasks.

In any way developing `build_recorder` is well worth it.

**Comment by samruddhibhoyar3** [comment]
I would like to contribute in the documentation
